examples/widgets/web_view.enaml

```diff
@@ -0,0 +1,27 @@
+#------------------------------------------------------------------------------
+# Copyright (c) 2018, Nucleic Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#------------------------------------------------------------------------------
+""" 
+
+This example shows how to use a WebView. 
+
+<< autodoc-me >>
+"""
+from enaml.widgets.api import Window, Container, WebView
+
+
+# Note: QtWebEngineWidgets must be imported before the QtApplication is started
```

Comment:
I would move/duplicate this to/in the docstring, to make it more visible in the documentation. Furthermore, is this also true under qt4 or is this qt5 only ?

Reply:
It appears to be with PyQt 5.10+. With 5.9.2 and the import is not needed.  With 5.10.1 and 5.11.2 and it is needed.

```python
Traceback (most recent call last):
  File "<root>/miniconda3/envs/occ/lib/python3.6/site-packages/qtpy/QtWebEngineWidgets.py", line 22, in <module>
    from PyQt5.QtWebEngineWidgets import QWebEnginePage
ImportError: QtWebEngineWidgets must be imported before a QCoreApplication instance is created
```

Should the version be checked and imported only if needed?

```
from enaml.qt.QtCore import __version_info__ as qt_version_info
if qt_version_info >= (5, 10):
    from enaml.qt import QtWebEngineWidgets
```